src/components/nav/ThemeToggler.tsx

```diff
@@ -0,0 +1,20 @@
+import {DarkModeSwitch} from '@/utils/DarkModeSwitch'
+import React from 'react'
+import {useLocalStorage} from './../hooks/useLocalStorage'
+
+export function ThemeToggler(): React.ReactElement {
```

Comment:
I don't think you need to specify the return type with `: React.ReactElement`. Seems redundant in this case.

Reply:
Removed